v2/integration/corpus_test.go

```diff
@@ -158,12 +158,17 @@ func TestCorpus(t *testing.T) {
 			*configFile)
 	} else {
 		// Otherwise enforce the maps match
+		failCounter := 0
 		for k, v := range resultsByLint {
 			if conf.Expected[k] != v {
 				t.Errorf("expected lint %q to have result %s got %s\n",
 					k, conf.Expected[k], v)
+				failCounter++
 			}
 		}
+		if failCounter > 0 {
+			t.Errorf("%d lint(s) failed", failCounter)
```

Comment:
Have you tried using `fmt.Printf` here instead? I believe stdout is eaten by the test runner except in the case where the test failed. It seems like a more natural way to display this information iff the test fails since it's presenting a summary and not reporting new error states.

Reply:
@cpu I followed your advise and updated the printing

Reply:
Thanks! Were the results still to your liking?

Reply:
yes, of course. The information is still printed out.

Reply:
Great 👍